ParkingZoneApp/Repository/ParkingSlotRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using ParkingZoneApp.Data;
+using ParkingZoneApp.Models.Entities;
+using ParkingZoneApp.Repository.Interfaces;
+
+namespace ParkingZoneApp.Repository
+{
+    public class ParkingSlotRepository : Repository<ParkingSlot>, IParkingSlotRepository
+    {
+        private readonly ApplicationDbContext _context;
+        public ParkingSlotRepository(ApplicationDbContext context) : base(context)
+        {
+            _context = context;
+        }
+
+        public ICollection<ParkingSlot> GetAllParkingSlots()
+        {
+            return _context.ParkingSlot.ToList();
+        }
```

Comment:
we dont need that instead of it create method in service called "GetSlotsById" that accepts zone id and returns slots belongs to specific parking zone